core/common-util/src/test/java/uk/gov/gchq/gaffer/commonutil/JsonUtilTest.java

```diff
@@ -13,94 +13,69 @@
-        final String json2 = "{\"a\": 1, \"b\": 2, \"c\": 3}";
-
-        assertFalse(JsonUtil.equals(json1, json2));
-        assertFalse(JsonUtil.equals(json1.getBytes(), json2.getBytes()));
-
-        JsonAssert.assertNotEqual(json1, json2);
-        JsonAssert.assertNotEqual(json1.getBytes(), json2.getBytes());
-    }
-
-    @Test
-    public void shouldReturnFalseWhenJsonObjectsAreNotEqual() {
-        final String json1 = "{\"a\": 1, \"b\": 2}";
-        final String json2 = "{\"a\": 1, \"b\": 3}";
-
-        assertFalse(JsonUtil.equals(json1, json2));
-        assertFalse(JsonUtil.equals(json1.getBytes(), json2.getBytes()));
-
-        JsonAssert.assertNotEqual(json1, json2);
-        JsonAssert.assertNotEqual(json1.getBytes(), json2.getBytes());
-    }
-
-    @Test
-    public void shouldReturnTrueWhenJsonArraysAreEqual() {
+    void shouldReturnTrueWhenJsonArraysAreEqual() {
         final String json1 = "[1,2,3]";
         final String json2 = "[1,2,3]";
 
-        assertTrue(JsonUtil.equals(json1, json2));
-        assertTrue(JsonUtil.equals(json1.getBytes(), json2.getBytes()));
+        assertThat(JsonUtil.equals(json1, json2)).isTrue();
+        assertThat(JsonUtil.equals(json1.getBytes(), json2.getBytes())).isTrue();
 
         JsonAssert.assertEquals(json1, json2);
         JsonAssert.assertEquals(json1.getBytes(), json2.getBytes());
     }
 
-    @Test
-    public void shouldReturnFalseWhenJsonArraysAreNotEqual() {
-        // Given
-        final String json1 = "[1,2,3]";
-        final String json2 = "[1,2,4]";
-
-        assertFalse(JsonUtil.equals(json1, json2));
-        assertFalse(JsonUtil.equals(json1.getBytes(), json2.getBytes()));
+    @ParameterizedTest
+    @MethodSource("provideJSONStrings")
+    void shouldReturnFalse(String json1, String json2) {
+        assertThat(JsonUtil.equals(json1, json2)).isFalse();
+        assertThat(JsonUtil.equals(json1.getBytes(), json2.getBytes())).isFalse();
 
         JsonAssert.assertNotEqual(json1, json2);
         JsonAssert.assertNotEqual(json1.getBytes(), json2.getBytes());
     }
 
-    @Test
-    public void shouldReturnFalseWhenJsonArraysAreDifferentSizes() {
-        final String json1 = "[1,2,3]";
-        final String json2 = "[1,2,3,4]";
-
-        assertFalse(JsonUtil.equals(json1, json2));
-        assertFalse(JsonUtil.equals(json1.getBytes(), json2.getBytes()));
-
-        JsonAssert.assertNotEqual(json1, json2);
-        JsonAssert.assertNotEqual(json1.getBytes(), json2.getBytes());
+    private static Stream<Arguments> provideJSONStrings() {
+        return Stream.of(
+            Arguments.of("{\"a\": 1, \"b\": 2}", "{\"a\": 1, \"b\": 2, \"c\": 3}"),
+            Arguments.of("{\"a\": 1, \"b\": 2}", "{\"a\": 1, \"b\": 3}"),
+            Arguments.of("[1,2,3]", "[1,2,4]"),
+            Arguments.of("[1,2,3]", "[1,2,3,4]")
+        );
     }
 
     @Test
-    public void shouldReturnFalseWhenActualObjectIsNull() {
+    void shouldReturnFalseWhenActualObjectIsNull() {
```

Comment:
Can this test be part of the parameterised shouldReturnFalse test? or can you not pass json2 as null?

Reply:
You can't pass json2 as null :)